src/TSHelper.ts

```diff
@@ -53,6 +56,51 @@ export function getExtendedType(node: ts.ClassLikeDeclarationBase, checker: ts.T
     return extendedTypeNode && checker.getTypeAtLocation(extendedTypeNode);
 }
 
+export function getExportable(exportSpecifiers: ts.NamedExports, resolver: EmitResolver): ts.ExportSpecifier[] {
+    return exportSpecifiers.elements.filter(exportSpecifier => isExportable(exportSpecifier, resolver));
+}
+
+export function isExportable(exportSpecifier: ts.ExportSpecifier, resolver: EmitResolver): boolean {
```

Comment:
It's not used anywhere except `getExportable`, so maybe it would be better to inline it?